public_html/wp-content/plugins/wordcamp-speaker-feedback/assets/js/script.js

```diff
@@ -72,6 +72,17 @@
 			} );
 	}
 
+	function characterCounter( event ) {
+		var len = event.target.value.length;
```

Comment:
This doesn't appear to handle multibyte characters, like emoji, the way I would expect. "💩" counts as two characters instead of one. On the PHP side we're using `mb_strlen` in the `validate_meta_string` callback to ensure that multibytes are only counted as one character, but I'm not sure how best to do that on the JS side.

Reply:
ah, of course it couldn't be this simple. i'll take another look in the morning, but if it still seems too complex I might pull this part out.

Reply:
Looks like there might be a solution [here](https://dmitripavlutin.com/what-every-javascript-developer-should-know-about-unicode/#33-string-length), although it might only be compatible with ECMAScript 2015.

Something like

```js
var len = [...event.target.value].length;
```

Reply:
Or there's this:
https://mathiasbynens.be/notes/javascript-unicode#accounting-for-astral-symbols

(Sorry, I got intrigued)

Reply:
Done in 34596f713ef7b03d937519a1ab473a93c9178a75